### ISSUE TITLE
CronJob's Last Schedule column value is missing the `aria-label` attribute

### ISSUE THREAD

**Issue by zerkms** [kubernetes/dashboard#3734]
##### Environment

```
Installation method: n/a
Kubernetes version: 1.14.1
Dashboard version: 1.10.1
Operating system: ubuntu
Node.js version ('node --version' output): n/a
Go version ('go version' output): n/a
```

##### Steps to reproduce

If you open a list of cron jobs either `Overview`, or `Workloads`, or `Cron Jobs` menu item you'd see that the `Last Schedule` column value is missing an `aria-label` attribute. Without it it's not possible quickly see when exactly the job was triggered last time.

**Comment by zerkms** [comment]
It's not valid for v2 anymore.
